src/main/java/Controller/musicEngineController.java

```diff
@@ -17,13 +17,12 @@ public class musicEngineController {
     private final int PLAYLIST = 1;
     private final int SPACE = 2;
```

Comment:
Is there a particular reason why we're giving each type a int representation rather than just using Strings?

Reply:
Cleaner to use constants for comparison + comparing strings is kinda futile in general